### DISCUSSION TITLE
Aerospace treating each tab as a separate application

### DISCUSSION THREAD

**Discussion by Asylcreek** [ghostty-org/ghostty#3913] [Q&A] [answered]
Hello.

So, I use Aerospace as my tiling manager of choice, and for reasons unknown, it seems to think that each Ghostty tab is a separate application when I set `macos-title-bar-style = tabs`.

I don't know if I am doing something wrong and this is the expected behaviour OR something is actually wrong.

**Check the top right corner of the menu bar for the Aerospace icon (what it looks like is in the image after the video)**

https://github.com/user-attachments/assets/3176b673-7c70-4e38-ac8b-bdca8cefe5e6

**icon on the top right corner to focus on**
<img width="74" alt="Screenshot 2024-12-29 at 18 08 45" src="https://github.com/user-attachments/assets/086acc83-3152-461c-b312-e624d07fcb65" />

My Ghostty config
```
font-family = Iosevka Italic
font-family-bold = Iosevka Oblique
font-family-italic = Iosevka Italic
font-family-bold-italic = Iosevka Oblique
font-size = 13
font-style = italic 
font-synthetic-style = italic 
font-thicken = false

theme = Adventure

background-opacity = 0.8
background-blur-radius = 20

window-decoration = true
window-inherit-font-size = true
window-title-font-family = Iosevka Italic
window-new-tab-position = current

macos-option-as-alt = true
macos-titlebar-style = tabs
macos-auto-secure-input = true
macos-secure-input-indication = true

cursor-style = underline
cursor-style-blink = true

mouse-hide-while-typing = true

clipboard-paste-protection = true
```

My machine is an M1 Pro Macbook Pro 16 2021 running Sequoia 15.2

**Comment by rhodes-b** [comment, marked as answer]
This is known please read here
https://ghostty.org/docs/help/macos-tiling-wms

**Reply by Asylcreek** [reply]
Thank you

**Comment by JakeWharton** [comment]
Dupe of https://github.com/ghostty-org/ghostty/discussions/3674?

**Reply by Asylcreek** [reply]
Maybe not ... but the information in the article shared above helped.

Thank you.
